doc/examples/Dockerfile.build

```diff
@@ -1,19 +1,91 @@
 # Build Debian package using dh-virtualenv
+#
+# To create a package for Stretch in `dist/`, call:
+#
+#   ./build.sh debian:stretch
 
-FROM #DIST_ID#:#CODENAME# AS dpkg-build
-ENV DEBIAN_FRONTEND=noninteractive
-RUN apt-get update -qq && apt-get install -yqq \
-        build-essential debhelper devscripts equivs dh-virtualenv \
-        curl tar gzip lsb-release apt-utils apt-transport-https libparse-debianchangelog-perl \
-        python3 python3-setuptools python3-pip python3-dev libffi-dev \
-        libxml2-dev libxslt1-dev libyaml-dev libjpeg-dev \
-        libssl-dev libncurses5-dev libncursesw5-dev libzmq3-dev \
-    && ( curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - ) \
-    && echo 'deb https://deb.nodesource.com/#NODEREPO# #CODENAME# main' \
+# Build arguments, as provided by 'build.sh'
+ARG DIST_ID="debian"
+ARG CODENAME="stretch"
+ARG PKGNAME
+
+# Other build arguments (adapt as needed)
+ARG NODEREPO="node_8.x"
```

Comment:
Whops, this is something that escaped me previously: where do we need the node repo?

Reply:
See the how-to right above it. the sample project I took simply needs it.

and it is always easier to remove stuff than to add it.

Reply:
I'd be bit hesitant to add a custom debian repo by default in people's builds. I do agree that it is easier to remove things than to add, but I'm not comfortable pushing a custom repo by default, when it is actually not needed for the build process itself.

Reply:
it is not obscure but nodesource, and these files are *tested* and working 1:1 copies out of a project.

Reply:
Yup, I understand that. I also don't think that particular custom debian repo is shady in any way. However this very documentation also ships with Debian (among others), and I'd like to keep it as lean as possible and only refer to sanctioned Debian repos.

It is really nice to have an example of the build file, but I'm wondering if we can find a simple example (which are, arguably, hard to fabricate) to fill the use case.

And down the line, people are going to just 1:1 copy these things and modify the few bits to fit their builds, so we could potentially introduce a pile of unnecessary steps in their build processes. Minor annoyance for sure, but still something